### ISSUE TITLE
Tags array doesn't prevent duplicates

### ISSUE THREAD

**Issue by AhmedSharaf98** [cviebrock/eloquent-taggable#66]
```
$model->tag('SomeTag , SomeTag');
$this->assertEquals(1 , $model->isTagged()->count());
```
This test fails : Failed asserting that 2 matches expected 1.
And it generates same result when passed as array also:
```
$model->tag(['SomeTag' , 'SomeTag']);
$this->assertEquals(1 , $model->isTagged()->count());
```
So they were considered two different tags, However making test like that passes
```
$model->tag('SomeTag);
$model->tag('SomeTag);
$this->assertEquals(1 , $model->isTagged()->count());
```

**Comment by cviebrock** [comment]
This is fixed in the 3.2.5 release.  Thanks!

**Comment by AhmedSharaf98** [comment]
Is there a fix for 3.1 for laravel 3.4?

**Comment by cviebrock** [comment]
Laravel 3.4?!

![](https://i.imgur.com/qvAlJmG.jpg)
